### DISCUSSION TITLE
Ability to adjust text alignment in Indicator panel

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#286] [Q&A]
From user request in slack:

> The only minor feedback I have is letting the user choose text alignment (left justify). I think it would look better if the button/bulb didn't keep shifting due to varying text length. I  guess the background mode is a good alternative.

**Comment by foxymiles** [comment]
@jtbandes Can you elaborate a bit more on what's being proposed here?

**Comment by jtbandes** [comment]
The text is currently centered. When the indicator switches to a different color/label, the colored circle moves because the text is a different length. Being able to left-align the text would fix this.
